src/i18n/langs/fr.json

```diff
@@ -29,7 +28,21 @@
   "Copyright": "Copyright",
   "Page Not Found": "Page non trouvée",
   "Approved by the International Year of Basic Sciences for Sustainable Development 2022": "Approuvé par l'Année internationale des sciences fondamentales pour le développement durable 2022",
-  "Energy balance": "Bilan énergétique",
-  "Global model including radiation, greenhouse gases, clouds and ice": "Modèle global incluant le rayonnement, les gaz à effet de serre, les nuages et les glaces",
-  "Brief tutorial of the simulations": "Bref tutoriel des simulations"
+  "GLOBAL_MODEL_TITLE": "Bilan énergétique",
```

Comment:
Should this be updated to the French equivalent of 'Global Model'?

Reply:
I think it's supposed to be the translation.